mappings/net/minecraft/util/math/ChunkSectionPos.mapping

```diff
@@ -136,6 +136,20 @@ CLASS net/minecraft/class_4076 net/minecraft/util/math/ChunkSectionPos
 		ARG 0 chunk
 	METHOD method_33706 toLong (Lnet/minecraft/class_2338;)J
 		ARG 0 pos
+	METHOD method_39489 forEachChunkSectionsAround (IIILit/unimi/dsi/fastutil/longs/LongConsumer;)V
+		COMMENT Performs an action for each chunk section enclosing a block position
+		COMMENT adjacent to {@code (x, y, z)}.
```

Comment:
```suggestion
		COMMENT adjacent or equal to {@code (x, y, z)}.
```

Reply:
On a second thought, just adjacent suffices.

Reply:
Don't think it checked for that exact coords. But shouldn't matter, chunk section is 16x16x16.